lib/std/compress/zlib.zig

```diff
@@ -1,14 +1,12 @@
-//
-// Compressor/Decompressor for ZLIB data streams (RFC1950)
-
+/// Compressor/Decompressor for ZLIB data streams (RFC1950)
```

Comment:
```suggestion
//! Compressor/Decompressor for ZLIB data streams (RFC1950)
```
Currently this is documenting the private `std` declaration, see https://ziglang.org/documentation/master/#toc-Top-Level-Doc-Comments.